numpy/core/numeric.py

```diff
@@ -3011,11 +3011,8 @@ class errstate(_errstate_baseclass):
     the context the error handling is set with `seterr` and `seterrcall`, and
     upon exiting it is reset to what it was before.
     
-    ..  versionchanged: 1.14.0
-        In Python 3, errstate is also usable as a function decorator, saving
-        a level of indentation if an entire function is wrapped.
-        
-        See :py:class:`contextlib.ContextDecorator` for more information.
+    ..  versionchanged: 1.14.0 
+        In Python 3, errstate is also usable as a function decorator (:py:class:`contextlib.ContextDecorator`)
```

Comment:
Really? I searched for `versionchanged` and all 3 uses are indented. I'll restore the extended description since https://github.com/numpy/numpy/blob/0767d38c8502aab52424ff9907ac1b63dc138597/numpy/ma/core.py#L5266 has something similar